### DISCUSSION TITLE
xml-sitemaps.com website give invalid sitemap

### DISCUSSION THREAD

**Discussion by arshadanjum167** [spatie/laravel-sitemap#390] [Q&A]
Hi
I have generate sitemap using this library and submit to https://www.xml-sitemaps.com/ and its result said that
"Incorrect http header content-type: "text/html" (expected: "application/xml")"

Please give me instruction if i am doing wrong.
sitemap : https://www.workversatile.com/sitemap.xml

**Comment by emtiazzahid** [comment]
any solution found?
